image/templates/helm/stackrox-central/templates/01-central-13-deployment.yaml

```diff
@@ -153,6 +157,10 @@ spec:
           mountPath: /run/secrets/stackrox.io/db-password
         - name: central-external-db-volume
           mountPath: /etc/ext-db
+        {{- if ._rox.central.encryptNotifierSecrets }}
```

Comment:
Is there a reason why you used the name  `NotifierSecretsEncryptionEnabled` in CR vs `encryptNotifierSecrets` in the helm chart. Would be simpler to search for and keep changes in sync if both are named the same.

Reply:
I was trying to follow the conventions in the surrounding code, but you're right, they should be called the same.

Reply:
Done.